apps/frontend/src/index.tsx

```diff
@@ -25,7 +26,7 @@ createRoot(document.querySelector("#root") as HTMLElement).render(
 							{
 								element: (
 									<ProtectedRoute
-										component="Root"
+										component={<QuizForm />}
```

Comment:
This shouldn't be in ROOT path. QuizForm will be removed in another ticket, which is in progress
Did you test the case, where the main content is too big?

Reply:
https://github.com/user-attachments/assets/346c9b76-3ca9-4e40-8a32-e4bf8f7ed5cf